packages/end-to-end/cypress/e2e/specs/01-onboarding.js

```diff
@@ -0,0 +1,35 @@
+describe.only('Onboarding', () => {
+  let users;
+  let translation;
+
+  beforeEach(() => {
+    // Load the users fixture before the tests
+    cy.fixture('e2e-test-users.json').then((loadedUsers) => {
+      users = loadedUsers;
+      const user = users[Cypress.env('USER')];
```

Comment:
Where would this env variable come from?

Reply:
This variable is coming from command line argument or CI workflow. Essentially it indicates which user from fixture to use. The intent is to call cypress command multiple times with different users (e.g. Spanish, English, etc)